windows-installer/template.nsi

```diff
@@ -60,7 +60,10 @@ SetCompressor lzma
 !insertmacro MUI_LANGUAGE "English"
 [% endblock modernui %]
 
-Name "${PRODUCT_NAME} ${PRODUCT_VERSION}"
+; CERTBOT CUSTOM BEGIN
+Name "${PRODUCT_NAME} (beta) ${PRODUCT_VERSION}"
```

Comment:
I prefer the former because I would like to avoid people thinking that it is the version that is a beta, while it is the concept of Certbot on Windows that is a beta.